File: programme/perimeter-of-rectangle/perimeter-of-rectangle.cpp
// Write a programme to find perimeter of rectangle

#include <iostream>
using namespace std;

int main()
{
    double len, wid;

    cout << "Enter the length : ";
    cin >> len;
    cout << "Enter the width  : ";
    cin >> wid;

    double perimeter = 2 * (len + wid);
    cout << "\nPerimeter of the rectangle: " << perimeter << endl;
    return 0;
}

File: programme/perimeter-of-rectangle/perimeter-of-rectangle.js
// Write a programme to find perimeter of rectangle

"use strict";

// perform following command to run programme
// node perimeter-of-rectangle.js

const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

rl.question("length  : ", (length) => {
  rl.question("breadth : ", (breadth) => {
    const p = (parseInt(length) + parseInt(breadth)) * 2;
    if (isNaN(p)) {
      console.error("Some error occured");
    } else {
      console.log(`\nperimeter : ${p}`);
    }
    rl.close();
  });
});
